src/Microsoft.Health.Dicom.Functions/Microsoft.Health.Dicom.Functions.csproj

```diff
@@ -18,11 +18,13 @@
     <PackageReference Include="Microsoft.Azure.WebJobs.Extensions.DurableTask" Version="2.4.3" />
     <PackageReference Include="Microsoft.NET.Sdk.Functions" Version="3.0.11" />
     <PackageReference Include="Newtonsoft.Json" Version="13.0.1" />
-    <PackageReference Include="System.Text.Encodings.Web" Version="4.5.1" PrivateAssets="All" />
+    <PackageReference Include="System.Text.Encodings.Web" Version="4.7.2" PrivateAssets="All" />
   </ItemGroup>
 
   <ItemGroup>
+    <ProjectReference Include="..\Microsoft.Health.Dicom.Blob\Microsoft.Health.Dicom.Blob.csproj" />
```

Comment:
we need `Microsoft.Health.Dicom.Metadata.csproj` instead. 
I made this same mistake before. :)